.gitignore.txt

```diff
@@ -0,0 +1,22 @@
+# Compiled class file
+*.class
+
+# Log file
+*.log
+forIgnore/
+
+# BlueJ files
```

Comment:
Please remove this section of ignoring (# BlueJ files and *.ctxt) as it doesn't related to Java